scss/variables/core/_magic-numbers.scss

```diff
@@ -118,6 +118,17 @@ $focus-shadow: 0 0 3px $color-cool-grey, 0 0 7px $color-cool-grey !default;
 //
 $nav-width: 951px !default;
 
+//
+// $breakpoint-navwidth
```

Comment:
This seems related to just the nav.. @josephgknox I cannot remember what we decided. All variables in the core file or can we have specifics in each mixin?